index.html

```diff
@@ -34,12 +34,236 @@ <h3>I am a software developer</h3>
             <div class="mess"><img src="./images/Icon - Medium.png" alt=""></div>
         </section> 
        </section>
-
-       <section>
-        <div>
-            <img src="./images/Icon - Angelist.png" alt="">
+       <section class="recent">
+        <div class="title">
+            <h2>My Recent works</h2>
+            <div class="v"></div>
+        </div>
+        <div class="frame">
+            <div class="featured">
+                <div class="imgp"></div>
+                <div class="block">
+                    <p class="post">Multi-Post Stories</p>
+                    <p class="support">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                    <section class="tag-main">
+                        <ul class="tag1">
+                            <li>css</li>
+                        <ul class="tag2">
+                            <li>html</li>
+                        </ul>
+                        <ul class="tag3">
+                            <li>bootstrap</li>
+                        </ul>
+                        <ul class="tag4">
+                            <li>Ruby</li>
+                        </ul>
+                    </section>
+                    <section class="buttom">
+                        <p class="see">See project</p>
+                    </section>
+                </div>
+            </div>
+            <section class="frame-sub">
+                <section class="card-works">
+                    <section class="mask">
+                        <section class="img-placeholder">
+                            <section class="div">
```

Comment:
- [ ] As the last code reviewer said: Please 🙏🏽 use an indentation of two spaces in all your documents. This is a convention and also [best practice.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)